firmware/common/portapack_persistent_memory.cpp

```diff
@@ -808,6 +812,14 @@ void set_config_freq_rx_correction(uint32_t v) {
     data->frequency_rx_correction = v;
 }
 
+// rotary encoder dial settings
+uint8_t config_encoder_dial_sensitivity() {
+    return data->encoder_dial_sensitivity;
+}
+void set_encoder_dial_sensitivity(uint8_t v) {
```

Comment:
Maybe bounds check before storing instead of on every read? I'm waffling on whether that's a good idea or not.